website/docs/components/link/standalone/partials/code/component-api.md

```diff
@@ -1,28 +1,28 @@
 ## Component API
 
 <Doc::ComponentApi as |C|>
-  <C.Property @name="size" @type="enum" @values={{array "small" "medium" "large" }} @default="medium"/>
-  <C.Property @name="color" @type="enum" @values={{array "primary" "secondary" }} @default="primary"/>
-  <C.Property @name="text" @required="true" @type="string">
-    The text of the link. If no text value is defined an error will be thrown.
-  </C.Property>
   <C.Property @name="icon" @required="true" @type="string">
-    Use this parameter to show an icon. Any [icon name](/foundations/icons) is acceptable. The `icon` is required to make the component accessible.
+    Use this parameter to show an icon. Any Helios [icon name](/foundations/icons/library) is acceptable. `icon` is required to ensure the component conforms to accessibility best practices.
   </C.Property>
   <C.Property @name="iconPosition" @type="enum" @values={{array "leading" "trailing" }} @default="leading">
     Positions the icon before or after the text.
   </C.Property>
+  <C.Property @name="color" @type="enum" @values={{array "primary" "secondary" }} @default="primary"/>
+  <C.Property @name="size" @type="enum" @values={{array "small" "medium" "large" }} @default="medium"/>
+  <C.Property @name="text" @required="true" @type="string">
+    The text of the link. If no text value is defined an error will be thrown.
+  </C.Property>
   <C.Property @name="href">
-    This is the URL parameter that is passed down to the `<a>` element.
+    URL parameter that’s passed down to the `<a>` element.
   </C.Property>
   <C.Property @name="isHrefExternal" @type="boolean" @values={{array "false" "true" }} @default="false">
-    This controls if the `<a>` link is external and so for security reasons we need to add the `target="_blank"` and `rel="noopener noreferrer"` attributes to it.
+    This controls if the `<a>` link is external. For security reasons, we add the `target="_blank"` and `rel="noopener noreferrer"` attributes to it by default.
```

Comment:
```suggestion
    Controls if the `<a>` link is external. For security reasons, we add the `target="_blank"` and `rel="noopener noreferrer"` attributes to it by default.
```
Same comment as badge (etc). I _could_ be debated on this as I'm reading through all of these lol